docs/openshift_horizontal_scaling.md

```diff
@@ -0,0 +1,105 @@
+# Openshift Horizontal Pod Autoscaling (HPA)
+
+It is possible to configure Apisonator to auto scale when deployed in Openshift.
```

Comment:
nit:
```suggestion
It is possible to configure Apisonator to automatically scale when deployed on Openshift.
```

Reply:
perhaps mention this can also be done in bare k8s?

Reply:
Done :+1: